js_modules/dagit/packages/core/src/assets/AssetNodeList.tsx

```diff
@@ -19,7 +19,8 @@ export const AssetNodeList: React.FC<{
       flex={{gap: 5}}
       padding={{horizontal: 12}}
       style={{
-        height: 112,
+        // height: 112,
+        minHeight: 112,
```

Comment:
I think this makes sense but do you know what you were looking to fix? I know at one point we picked these heights so that the right side of the definition UI would align with the height of the collapsed description on the left, but it's less necessary now that we're moving it to a separate tab

Reply:
After experimenting a bit, I've reverted it, I think that was just baggage I forgot to delete when I was working on the new "Type" section and how to align it.